README.md

```diff
@@ -258,6 +272,17 @@ ansible-playbook migrate-db.yml                                    \
 
 It's safe to run migrations even if there are no changes - Django will detect that and simply leave the schema as is.
 
+#### Helper script for deployment process
+
+###### `build-and-deploy-to-dev.sh`
+This script is available on the remote `concent-builder` server.
+Before connect to server and run script, you must locally specify version of concent in the `containers/versions.yml` file and push it as commit to dev branch on the `concent-deployment` repository.
+
+Description of the script:
+- configures environment for deployment process in your home directory. You can also specify custom path by second argument in the script.
+- builds and deploys specifics concent version on the dev cluster.
+- migrates or resets databases, default is migrate. To reset databases use `reset` as first argument in the script.
```

Comment:
Please reword:
``` markdown
The script performs its job by running the Ansible playbooks listed above in the right order and with the parameters suitable for the `concent-dev` cluster.

Command-line options:
1. **database_operation**: _migrate_ (default) or _reset_.
    _migrate_ does not clear the database runs the migrations.
    _reset_ clears the database and reinitializes it.
2. **deployment_dir**: the directory to store a clone of the `concent-deployment` repository.
    `~/deployment` by default.
    This working copy should not be modified manually.
    The script will recreate it if it does not exist or update it if it does but it will fail if a checkout is not possible (e.g. because there are uncommitted changes).
```